presto-hive/src/test/java/com/facebook/presto/hive/TestHivePushdownFilterQueries.java

```diff
@@ -37,15 +37,17 @@
     private static final Pattern ARRAY_SUBSCRIPT_PATTERN = Pattern.compile("([a-z_+]+)((\\[[0-9]+\\])+)");
 
     private static final String WITH_LINEITEM_EX = "WITH lineitem_ex AS (\n" +
-            "SELECT linenumber, orderkey, partkey, suppkey, \n" +
+            "SELECT linenumber, l.orderkey, partkey, suppkey, \n" +
```

Comment:
Is there any particular reason to introduce a join? There are 3 date columns in lineitem: shipdate, commitdate, receiptdate. You can convert 2 of these or all 3 to timestamp.

Reply:
wanted to compare shipdate vs orderdate to keep it more meaningful

Reply:
Is it ok to keep the join ?